### ISSUE TITLE
Missing link to the original research project

### ISSUE THREAD

**Issue by yonatanp** [nitaybz/homebridge-electra-smart#7]
Great initiative, good to see the API used for a homebridge client.
Looks like credit is due to the [electrasmart](https://github.com/yonatanp/electrasmart)  project where the original research came from.
Beyond the common decency of giving credit for hard work being reused, it would also be helpful for future developers to know that there are other implementations to use as reference. For example, Jafar Atili’s python implementation.
I would be happy to send a PR if that helps.

**Comment by nitaybz** [comment]
Fair point. Adding a credit / inspiration link to the README is the right thing to do. I'll do that as part of a documentation pass at some point. For the record, the original electra-smart protocol research that this plugin builds on can be found across various community write-ups, including the original Python implementations. If you have a specific repo in mind that you want credited, post the link and I will add it explicitly.

Sorry for the long delay on this. Closing this for now since it is not a runtime issue.

**Comment by yonatanp** [comment]
Never mind, don’t worry about it, life is short, credit is not important,
go to the good things
